wifiphisher/common/opmode.py

```diff
@@ -264,7 +293,18 @@ def freq_hopping_enabled(self):
         """
 
         return self.op_mode in [constants.OP_MODE1,
-                                constants.OP_MODE2]
+                                constants.OP_MODE2,
+                                constants.OP_MODE7]
+
+    def wps_assoc_enabled(self):
+        """
+        :param self: An OpMode object
+        :type self: OpMode
+        :return: True if we define the wps associating interface
+        :rtype: bool
+        """
+        return self.op_mode in [constants.OP_MODE7,
+                                constants.OP_MODE8]
```

Comment:
So this should also change to "assoc_enabled".

And ":return: True if we are using managed Extentions (that associate to WLANs)".